src/main/g8/build.sbt

```diff
@@ -0,0 +1,61 @@
+// FIXME once akka 2.6.9 is released
+val AkkaVersion = "2.6.8+71-57fb9e90"
+val AkkaPersistenceCassandraVersion = "1.0.1"
+val AlpakkaKafkaVersion = "2.0.4"
+val AkkaHttpVersion = "10.2.0"
+// FIXME once akka management 1.0.9 is released
+val AkkaManagementVersion = "1.0.8+35-9feaa689+20200825-1429"
+val AkkaProjectionVersion = "1.0.0-RC1"
+
+enablePlugins(AkkaGrpcPlugin)
+
+name := "$name$"
+version := "1.0"
+
+organization := "com.lightbend.akka.samples"
+organizationHomepage := Some(url("https://akka.io"))
+licenses := Seq(("CC0", url("https://creativecommons.org/publicdomain/zero/1.0")))
+
+// For akka management snapshot
+resolvers += Resolver.bintrayRepo("akka", "snapshots")
+// For akka nightlies
+resolvers += "Akka Snapshots" at "https://repo.akka.io/snapshots/"
+
+Compile / scalacOptions ++= Seq("-deprecation", "-feature", "-unchecked", "-Xlog-reflective-calls", "-Xlint")
+Compile / javacOptions ++= Seq("-Xlint:unchecked", "-Xlint:deprecation")
+
+Test / parallelExecution := false
+Test / testOptions += Tests.Argument("-oDF")
+Test / logBuffered := false
+
+scalaVersion := "2.13.3"
+
+libraryDependencies ++= Seq(
```

Comment:
I'm a big fan of moving anything regarding dependencies to `project/Dependencies.scala`.

This small case, in particular, has a lot of unnecessary complexity due to snapshots, snapshot resolver, etc... and a rather long list of artifact versions (lines 1-9). Moving all that away will make this `build.sbt` less scary.

Reply:
Created issue https://github.com/akka/akka-platform-guide/issues/124